### PR TITLE
added table example

### PR DESCRIPTION
**Pull request by 007tom**
**Summary**

1. added a table example with React-Hooks
2. supported table insertion
3. and supported table cell edit operation

**Test Plan**

[...]

### CODE REVIEW

**Comment by aomini** [comment]
Awesome man.

File: examples/draft-0-11/table/src/components/RichTextExample.js [hunk 1]
```diff
@@ -1,72 +1,84 @@
...
+    if (block.getType() === 'atomic') {
+      return {
+        component: BlockComponent,
+        editable: false,
+        props: {
+          onStartEdit: blockKey => {
+            setTableEdits(tableEdits.set(blockKey, true));
+          },
+          onFinishEdit: blockKey => {
+            setTableEdits(tableEdits.remove(blockKey));
```
Comment (aomini) [inline examples/draft-0-11/table/src/components/RichTextExample.js:28]: On Finish Edit the editor still stays on read only mode.  Do we have any other workaround?

Comment (007tom) [reply examples/draft-0-11/table/src/components/RichTextExample.js:28]: The feature is just experimental, 
To be honest, it is just a toy, and
now I have no time to develop it in this month or longer, 
you can try the website： https://zhihu.com/, it also uses draft-js and has implemented table feature.

Comment (aomini) [reply examples/draft-0-11/table/src/components/RichTextExample.js:28]: Anyway thanks. Your PR saved me a lot of time.

**Pull request closed without merging**